src/seedsigner/gui/components.py

```diff
@@ -190,6 +190,8 @@ class FontAwesomeIconConstants:
     SQUARE_CARET_UP = "\uf151"
     UNLOCK = "\uf09c"
     X = "\u0058"
+    INDEX = "\u0023"
```

Comment:
Font Awesome refers to this as "hashtag". I usually refer to it as "pound", though I think that might be a little outdated. I'd probably go with `HASHTAG` here.